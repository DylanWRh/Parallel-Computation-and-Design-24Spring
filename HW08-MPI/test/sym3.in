8 -3 -3 
-6 -11 
18 
