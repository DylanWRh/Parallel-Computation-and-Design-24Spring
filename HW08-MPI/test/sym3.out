Eigenvalues:
-11.0732
8.5863
22.4869

Eigenvalues
0.200389 0.97223 -0.120888 
0.9093 -0.230499 -0.346473 
0.364716 0.0404942 0.930238 
